app/src/main/res/layout/activity_login.xml

```diff
@@ -2,92 +2,107 @@
-                android:layout_width="match_parent"
-                app:passwordToggleEnabled="true">
-
-                <EditText
-                    style="@style/Base.TextAppearance.AppCompat.Medium"
-                    android:drawableLeft="@drawable/ic_wrapped_lock_black_24dp"
-                    android:drawablePadding="@dimen/layout_padding_8dp"
-                    android:drawableStart="@drawable/ic_wrapped_lock_black_24dp"
-                    android:hint="@string/password"
-                    android:inputType="textPassword"
+                </com.google.android.material.textfield.TextInputLayout>
+
+                <com.google.android.material.textfield.TextInputLayout
+                    android:id="@+id/til_password"
+                    style="@style/TextInputLayoutLogin"
+                    android:layout_width="match_parent"
                     android:layout_height="wrap_content"
-                    android:layout_width="match_parent"/>
-            </com.google.android.material.textfield.TextInputLayout>
+                    android:backgroundTint="@color/white"
+                    android:textColorHint="@color/white"
+                    app:passwordToggleEnabled="true"
+                    app:passwordToggleTint="@color/white">
+
+                    <com.google.android.material.textfield.TextInputEditText
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:fontFamily="@font/productsans"
+                        android:inputType="textPassword"
+                        android:textColor="@color/white"
+                        android:textColorHighlight="@color/white"
+                        android:hint="@string/password" />
+
+                </com.google.android.material.textfield.TextInputLayout>
 
-            <androidx.appcompat.widget.AppCompatButton
+            </LinearLayout>
+
+            <com.google.android.material.button.MaterialButton
                 android:id="@+id/btn_login"
-                android:layout_height="wrap_content"
-                android:layout_marginBottom="@dimen/default_margin"
-                android:layout_marginTop="@dimen/login_margin_top"
-                android:layout_width="match_parent"
+                android:layout_gravity="center"
+                android:layout_height="@dimen/layout_padding_50dp"
+                android:layout_marginTop="@dimen/layout_padding_30dp"
+                android:layout_width="@dimen/layout_padding_100dp"
                 android:text="@string/login"
-                android:textColor="@color/white"/>
+                android:backgroundTint="@color/white"
+                android:textColor="@color/grey_redesign"
+                android:fontFamily="@font/productsans"/>
 
-            <androidx.appcompat.widget.AppCompatButton
-                style="@style/Widget.AppCompat.Button.Borderless"
+            <TextView
```

Comment:
Why it's changed to `textview`? if it's needed, let's rename `id` properly.

Reply:
I'll change the id to `tv_register`